verification/certs.go

```diff
@@ -3,11 +3,116 @@
 package verification
```

Comment:
Functions that validate various fields are moved here from certifyKey.go. This is an effort to keep helper functions that are independent of DPEClient instance in certs.go
This was done to reduce the bloating in CertifyKey.go